### ISSUE TITLE
How to roll my own ODH deployment and work on Elyra Pipelines?

### ISSUE THREAD

**Issue by pacospace** [operate-first/support#208]
**Is your feature request related to a problem? Please describe.**
As user interested in ODH and Elyra,

I would like to read the documentation on how to roll out my own ODH deployment with some basic functionalities and read some basic tutorials about using Jupyterlab/Elyra and creating pipelines.

**Describe the solution you'd like**
A basic documentation and description specifically for:

- [ ] Overview of the components considered.
- [ ] ODH setup basic functionalities guide (ready to use manifest files included that only install the bare minimum) (e.g. JupyterHub deployment with JupyterLab/Elyra (to work with notebooks interactively or in pipelines)
- [ ] Kubeflow installation to be used for Elyra pipeline
- [ ] several tutorials to familiarize the audience with concepts (e.g. https://www.operate-first.cloud/users/elyra/README.md and more from Elyra team)

In summary, would be nice to have a brief overview of the components, guides through the installation (ready to use manifest files included that only install the bare minimum), and several tutorials to familiarize the audience with concepts.

The proposal is from: @ptitzler. Thank you!

**Describe alternatives you've considered**

**Additional context**
End goal: Possibly improve documentation upstream for ODH.

cc @tumido @durandom @vpavlin

**Comment by ptitzler** [comment]
For reference, this is what we currently have in the Elyra documentation: https://elyra.readthedocs.io/en/latest/recipes/deploying-elyra-with-opendatahub.html. Our goal is to completely eliminate the need for ODH deployment instructions in the Elyra documentation and link instead to the appropriate topic in the ODH documentation. If I had to redo this doc, it'd probably cover:
- install ODH operator
- use operator to deploy Kubeflow 
- use operator to deploy JupyterHub
- how to access JupyterLab/Elyra image
- how to create runtime configuration for Kubeflow instance
- link to hello world pipeline tutorial

**Comment by tumido** [comment]
I'm not sure how much does this fit under Operate First, but there's some kind of ready docs on ODH installation and so on:

- [Install ODH](https://opendatahub.io/docs/getting-started/quick-installation.html)
- [Deploy Kubeflow via ODH](https://opendatahub.io/docs/kubeflow/installation.html)
- [Adding custom images](https://opendatahub.io/docs/administration/installation-customization/add-custom-image.html) e.g. Elyra image

And @pacospace's docs on runtime configuration:
- [Create runtime via Kubeflow](https://github.com/thoth-station/elyra-aidevsecops-tutorial/blob/master/docs/source/create-ai-pipeline.md)

The whole repo is basically a demo of a Elyra via Kubeflow on ODH - I think that counts as an "advanced" hello world:
https://github.com/thoth-station/elyra-aidevsecops-tutorial/#tutorial-steps

Sure we may need to consolidate those docs somehow, but I'm not sure where's the right place for it to live. I don't think Operate First should maintain workload specific user documentations... We should definitely link to it though. :slightly_smiling_face:
